api/services/EntranceService.js

```diff
@@ -396,14 +396,22 @@ module.exports = {
     if (!entrance) return null;
 
     if (entrance.cave) {
-      [entrance.cave.massifs, entrance.cave.entrances] = await Promise.all([
-        CaveService.getMassifs(entrance.cave.id),
-        TEntrance.find({ cave: entrance.cave.id }),
-      ]);
-      await Promise.all([
-        NameService.setNames(entrance.cave.massifs, 'massif'),
-        NameService.setNames([entrance.cave], 'cave'),
-      ]);
+      const cave = await TCave.findOne(entrance.cave.id)
```

Comment:
Why fetching the `cave` again ? 
It was already fetched with the `.populate('cave')` and `author` and `reviewer` won't be retained in `toSimpleCave()`